### DISCUSSION TITLE
Unable to click checkboxes in issue due to console error on page load

### DISCUSSION THREAD

**Discussion by hepcat72** [community/community#20389] [Projects and Issues]
When I go to an issue on our github, when the page loads, the console shows this error:

```
Unrecognized Content-Security-Policy directive 'worker-src'.
```

and if I click a checkbox in the issue, the console shows this error:

```
Failed to load resource: the server responded with a status of 422 ()
```

relating to the issue page URL.

Is this an issue with my browser that I can fix or is it a problem with github?  Everything is green on the [status](https://www.githubstatus.com) page.

*UPDATE:* Must be a browser issue? When I load the page in Chrome, the status of the assortment of checkboxes on the page differs from in Safari.  I tried a shift-reload in Safari to force all resources to load, but I still get all the same errors.  Chrome's console does have a warning:

```
Error with Permissions-Policy header: Origin trial controlled feature not enabled: 'interest-cohort'.
```

Perhaps that's relevant to what is happening in Safari?

**Comment by diegoleondev** [comment]
👋 Hi @hepcat72

It's a Safari issue, but don't worry, it's not a big deal. You can read a bit more in this [MDN article](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/frame-ancestors#browser_compatibility).

>**Note**
>Please **Mark as answer** if it was helpful.
